src/material.js

```diff
@@ -1,7 +1,9 @@
-import shaderSources from './gl/shader_sources'; // built-in shaders
 import GLSL from './gl/glsl';
 import {StyleParser} from './styles/style_parser';
 
+let fs = require('fs');
```

Comment:
It feels kind of weird to be mixing node `require` along with ES6 `import`, no? (Babel just transpiles the imports to requires.)